openapi/openapi.yaml

```diff
@@ -247,3 +247,85 @@ paths:
           description: The current User
         'default':
           description: Error
+components:
+  schemas:
+
+    PageCount:
+      description: 'The number of items in this Page'
+      type: integer
+      example: 5
+
+    PageInfo:
+      description: 'Pagination info'
+      title: 'Page Info'
+      properties:
+        meta:
+          $ref: '#/components/schemas/PageMeta'
+        links:
+          $ref: '#/components/schemas/PageLinks'
+      required:
+        - meta
+        - links
+
+    PageLinks:
```

Comment:
I agree that there's no need to extract *PageCount*. Wondering why we're extracting *PageLinks*. Can we combine all the properties into *PageInfo* and make this simpler.

Reply:
Having standalone PageLinks might be reasonable, depending on how referencing this schema from endpoints.

Reply:
The JSON API spec that the insights pagination is based on considers 'links' it's own object (https://jsonapi.org/format/#document-links) so following that convention.

It is also how other insights apps model it:
https://github.com/RedHatInsights/vulnerability-engine/blob/master/manager.spec.yaml#L611
https://github.com/RedHatInsights/insights-rbac/blob/master/docs/source/specs/openapi.json#L1214